### PR TITLE
Changes to ensure that single repo can handle both bibs and items

### PR DESCRIPTION
**Pull request by jmathewt**
Created an EnvironmentConfig file to handle all the System.getenv() variables. 
Added an additional environment variable processBib (expecting true or false) values.

### CODE REVIEW

File: src/main/java/org/nypl/harvester/sierra/model/streamdatamodel/SierraResourceRetrievalRequest.java [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
-import org.nypl.harvester.sierra.model.Item;
+import org.nypl.harvester.sierra.model.Resource;
 import org.nypl.harvester.sierra.model.StreamDataModel;
 
-public class SierraItemUpdate extends StreamDataModel {
+public class SierraResourceRetrievalRequest extends StreamDataModel {
   private String id;
 
   @Override
   public boolean translateToStreamData(Object data) {
```
Comment (kfriedman) [inline src/main/java/org/nypl/harvester/sierra/model/streamdatamodel/SierraResourceRetrievalRequest.java:10]: Can we make `Object` more specific to `Resource`?

**Review by kfriedman — APPROVED** [review]
Looks good Jobin. Thanks for doing this. Just one minor suggestion about the `Object` reference.

File: src/main/java/org/nypl/harvester/sierra/model/streamdatamodel/SierraResourceRetrievalRequest.java (hunk 1 above)
Comment (jmathewt) [reply src/main/java/org/nypl/harvester/sierra/model/streamdatamodel/SierraResourceRetrievalRequest.java:10]: Change made and committed to branch - improve_kinesis_posting_logic

**Comment by jmathewt** [comment]
improve_kinesis_posting_logic branch has been opened for PR with latest changes. Closing this.

**Pull request closed without merging**